k8s-rule.yaml

```diff
@@ -0,0 +1,1034 @@
+- message: Deprecated/removed Kubernetes API version 'extensions/v1beta1' is used for 'Deployment'. Consider using 'apps/v1'.
```

Comment:
Can these rules be generated or did you do this by hand?

Reply:
nah it's not generated

Reply:
I think we can find a way to get this automated.

Reply:
Wow, that must have taken a long time, thank you. This is going to be a nice feature!

Reply:
trust me, that didn't take much time :sweat_smile: